### PR TITLE
Add DELETE method to /operators/:id

### PR DESCRIPTION
**Pull request by paniagua**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1135,6 +1135,37 @@ To create a new operator, you must provide a unique email address.
...
+
++ Response 200 (application/json)
+
+           {
+             "href": "https://api.salemove.com/operators/6",
+             "name": "John Doe",
+             "email": "test@test.com",
+             "phone": "2980773123",
+             "role": "operator",
+             "available": true
```
Comment (stenver) [inline apiary.apib]: We should probably add `enabled` to this list for this and GET

Comment (indrekj) [inline apiary.apib]: remove `available`, this is going to change or be removed anyway. btw, do we need to send a record for DELETE? Isn't 200 enough?

File: apiary.apib [hunk 2]
```diff
@@ -1135,6 +1135,37 @@ To create a new operator, you must provide a unique email address.
...
+
+            Authorization: Token 2pFuPlTYkH7T4MylrwxApA
+            Accept: application/vnd.salemove.v1+json
+
+    + Body
+
+                {
+                }
+
++ Response 200 (application/json)
```
Comment (indrekj) [inline apiary.apib]: 204

**Comment by paniagua** [comment]
Updated

File: apiary.apib [hunk 3]
```diff
@@ -1135,6 +1135,31 @@ To create a new operator, you must provide a unique email address.
...
+
+    + Body
+
+                {
+                }
+
++ Response 204 (application/json)
+
+           {
+           }
```
Comment (indrekj) [inline apiary.apib]: Remove these two lines

File: apiary.apib [hunk 4]
```diff
@@ -1135,6 +1135,31 @@ To create a new operator, you must provide a unique email address.
...
+
+            Authorization: Token 2pFuPlTYkH7T4MylrwxApA
+            Accept: application/vnd.salemove.v1+json
+
+    + Body
+
+                {
+                }
+
++ Response 204 (application/json)
```
Comment (indrekj) [inline apiary.apib]: Remove `(application/json)`

File: apiary.apib [hunk 5]
```diff
@@ -1135,6 +1135,31 @@ To create a new operator, you must provide a unique email address.
...
+To disable an operator you must provide the operator's id.
+
++ Request
+
+    + Headers
+
+            Authorization: Token 2pFuPlTYkH7T4MylrwxApA
+            Accept: application/vnd.salemove.v1+json
+
+    + Body
```
Comment (indrekj) [inline apiary.apib]: I guess we don't need body section if we don't have anything here

**Comment by indrekj** [comment]
Few minor comments, otherwise +1

File: apiary.apib [hunk 6]
```diff
@@ -1135,6 +1135,31 @@ To create a new operator, you must provide a unique email address.
              "available": true
            }
 
+
+### Disable anoperator [DELETE]
```
Comment (deiwin) [inline apiary.apib]: anoperator

File: apiary.apib [hunk 7]
```diff
@@ -1135,6 +1135,31 @@ To create a new operator, you must provide a unique email address.
...
+
+The manager can disable an operator from the SaleMove platform.
```
Comment (deiwin) [inline apiary.apib]: 'A manager', perhaps

File: apiary.apib [hunk 8]
```diff
@@ -1135,6 +1135,31 @@ To create a new operator, you must provide a unique email address.
...
+
+To disable an operator you must provide the operator's id.
```
Comment (willmore) [inline apiary.apib]: ID should be capitalized.

**Comment by willmore** [comment]
Hey all - what's status on this? Getting might stale.

**Comment by paniagua** [comment]
updated

File: apiary.apib [hunk 9]
```diff
@@ -1135,6 +1135,22 @@ To create a new operator, you must provide a unique email address.
              "available": true
            }
 
+
+### Disable an operator [DELETE]
```
Comment (deiwin) [inline apiary.apib]: I think this should be under the `## Operator [/operators/{operator_id}]` section instead. Doesn't really make sense here.

Comment (paniagua) [inline apiary.apib]: moved

**Comment by deiwin** [comment]
+1

**Comment by stenver** [comment]
!squash

**Pull request merged**